### DISCUSSION TITLE
How to create a form that shows file preview based on selected data.

### DISCUSSION THREAD

**Discussion by bumpyy** [filamentphp/filament#6588] [Help] [answered]
So I have this setting page (I'm using the spatie plugin) that has a dropdown that list a bunch of movies,  I want to show the poster and trailer in the form like file upload preview  based on the movie selected via said dropdown so admin can have preview before save.

Let's just say I already got a way to get the movie id, but haven't found a way to preview the file.

Anyone got any suggestions ?

**Comment by danharrin** [comment, marked as answer]
You can use a `View::make('x')` component. Inside, you can get the value of another field in PHP using `$evaluate(fn ($get) => $get('field_name'))`

**Reply by bumpyy** [reply]
Thanks, I just reading about custom field, this answer helpful because now I know what to search for, sometime there is something that isn't explained fully for newbie like me in the doc, it's a really good doc though.
